actors/runtime/src/main/java/cloud/orbit/actors/runtime/Hosting.java

```diff
@@ -254,10 +260,11 @@ private void onClusterViewChanged(final Collection<NodeAddress> nodes)
         consistentHashNodeTree = newHashes;
         updateServerNodes();
 
+        Collection<NodeAddress> values = localAddressCache.asMap().values();
         for (NodeInfo info : oldNodes.values())
         {
             // clear local cache
-            localAddressCache.values().remove(info.address);
+            values.remove(info.address);
```

Comment:
Is this a bug? The `values` is a collection of `NodeAddress` but it doesn't mean there is only one that matches as the localAddressCache is a collection of `RemoteReference` to `NodeAddress`. My understanding is that multiple keys can point to the same value. We would be wanting to remove all actor references from the localAddressCache for the oldNodes of Orbit?